test/scale/test-scale.sh

```diff
@@ -406,6 +427,7 @@ set -x
 $KUBECTL $KUBECONFIG_ARG create ns scale-test
 $KUBECTL $KUBECONFIG_ARG apply -f generated/kwok-nodes/
 $KUBECTL $KUBECONFIG_ARG apply -f generated/deployments/real/
+$KUBECTL $KUBECONFIG_ARG apply -f generated/services/real/
```

Comment:
We'll need to apply only if `numServices > 0`. Like here:
https://github.com/Azure/azure-container-networking/pull/1975/files/24467195c5ac9edd90793a3befd95ebe029098f4#diff-7f8f18cf7c38d320fefc80745a20c806a8b4f626e8fb27bf7d009a0daf96de4f

Reply:
Cool, will add it.